lib/nrf_modem_lib/Kconfig.modemlib

```diff
@@ -18,6 +18,10 @@ config NRF_MODEM_LIB_SYS_INIT
 	  Please note that initialization is synchronous and can take up to one
 	  minute in case the modem firmware is updated.
 
+config NRF_MODEM_TRACE_IRQ_PRIORITY
+	int "IRQ priority for trace handler"
+	default 6
```

Comment:
Would it make sense to set a different default in case ZLI is enabled, as 6 is not a valid one as you say?

Reply:
An irq priority of 4 is compatible when ZLI is enabled. I suggest to make this the default value, than no intervention is necessary when ZLI is enabled. I feel uncomfortable to "automatically" change the priority based on CONFIG_ZERO_LATENCY_IRQS as this adds more "magic" than necessary.